src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/AdvancedSettingsModal.js

```diff
@@ -0,0 +1,60 @@
+import { Help } from '@1hive/1hive-ui'
+import React, { Fragment, useCallback, useState } from 'react'
+import { useOnboardingState } from '@/providers/Onboarding'
+import { Modal, PercentageField } from '@components/Onboarding//kit'
+import MaxConvictionActiveTokensChart from './Charts/MaxConvictionActiveTokensChart'
+
+const CHART_HEIGHT = '350px'
+const CHART_WIDTH = '700px'
+
+const AdvancedSettingsModal = ({ stakeOnProposalPct, visible, onClose }) => {
+  const { config, onConfigChange } = useOnboardingState()
+  const [minThresholdStakePct, setMinThresholdStakePct] = useState(
+    config.conviction.minThresholdStakePct
+  )
+
+  const handleMinThresholdStakePct = useCallback(
+    value => {
+      setMinThresholdStakePct(value)
+    },
+    [setMinThresholdStakePct]
+  )
+
+  const handleDoneClick = () => {
+    onConfigChange('conviction', { ...config.conviction, minThresholdStakePct })
```

Comment:
Yes, I agree with that. At the time thought it was best to do it this way to avoid using a redundant inner state variable `minThresholdStakePct`. 

It's better to keep consistency though